CHANGELOG.md

```diff
@@ -12,16 +12,6 @@ Release date: TBD
 
 ### Improvements
 
-#### All Platforms
- - Possible improvement when the app is in background state.
- [#563](https://github.com/mattermost/desktop/pull/563)
-
-#### Windows
- - [Windows 7/8] Added support to open the message when clicking desktop notification.
- [#67](https://github.com/mattermost/desktop/issues/67)
- - Added support for different DPI across monitors.
- [#357](https://github.com/mattermost/desktop/issues/357)
-
 ### Bug Fixes
```

Comment:
Should we remove these section headers since they don't have any content in them?

Reply:
Not necessary, but either one is fine. It would have some items in future, and we can remove it if there are no items when cutting a release.

Reply:
0/5, we can, but that section is for the next version that's `under development`.

Note that the bug fixes section does have a few items, grouped to `all platforms` vs `windows` vs ...